src/js/strategies/CitrixVDIStrategy.js

```diff
@@ -0,0 +1,131 @@
+import CCPInitiationStrategyInterface from "./CCPInitiationStrategyInterface";
+import {hitch, isCitrixWebRTCSupported} from "../utils";
+import {FailedState} from "../rtc_session";
+import {RTC_ERRORS} from "../rtc_const";
+import "@citrix/ucsdk/CitrixWebRTC";
+
+export default class CitrixVDIStrategy extends CCPInitiationStrategyInterface {
```

Comment:
are you planning to add unit test for these 2 strategy classes?

Reply:
Yeah, will add more unit test causes to cover these.